external/model-preparation-algorithm/mpa_tasks/apis/task.py

```diff
@@ -181,6 +184,34 @@ def _initialize(self, dataset=None, output_model=None, export=False):
                 del self._model_cfg['fp16']
         self._precision = [ModelPrecision.FP32]
 
+        # Add/remove adaptive interval hook
+        if 'use_adaptive_interval' in self._recipe_cfg:
+            if self._recipe_cfg.use_adaptive_interval is True:
+                ada_interval_cfg = self._recipe_cfg.get('adaptive_validation_interval', None)
+                if ada_interval_cfg is None:
+                    self._recipe_cfg.adaptive_validation_interval = dict(max_interval=5)
```

Comment:
Doesn't this cause an error if `adaptive_validation_interval `does not exist?
and If it's not an issue that occurs, it seems that `self._recipe_cfg.adaptive_validation_interval = self._recipe_cfg.get('adaptive_validation_interval', dict(max_interval=5))` is more appropriate.

Reply:
Doesn't it just overwrite to its own value? Can you explain why it's more appropriate? And do you think it causes error? I'm just curious.

Reply:
I was just thinking about maybe we could reduce the code to three lines -> 1 line code. Also I thought there would be an issue if `self._recipe_cfg` didn't have `adaptive_validation_interval`.

Reply:
I modified this part. Please check it again!